common/lib/capa/capa/capa_problem.py

```diff
@@ -285,7 +285,9 @@ def is_optioninput_valid(optioninput):
             correct_option = None
             child_options = []
             for option_element in optioninput.findall('./option'):
-                option_name = option_element.text.strip()
+                text = option_element.text
```

Comment:
Question: So the offending problem options with no text would be `None` here - correct?

Reply:
Yeah, the `text` is coming through as `None`, so the bug manifests when trying to call `.strip()` on what it _expects_ is a string, but is actually `None`.